mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -643,6 +643,12 @@ impl Nexus {
 
         match errno_result_from_i32((), errno) {
             Ok(_) => {
+                // spdk_bdev_register registers a poller to wait for the
+                // examine to complete before calling spdk_notify_send with a
+                // pointer to the bdev name, which could be gone if the bdev
+                // was destroyed too soon after creation.
+                // Poll a few times to have that run.
+                Reactors::current().poll_times(3);
```

Comment:
Did you pick this by fair dice roll?

Reply:
Actually I iteratively increased the number until the core test passed :) You could contrive another test that'll make it fail again. I think this'll do as a temporary band-aid until an SPDK-only reproducer can be created and then reported upstream.

Reply:
This code will go away once we switch to our controller-based devices.

Reply:
How likely is this to be an issue in kubernetes? and would it be worth increasing the number of polls to get some wiggle room?

Reply:
This affects very short-lived bdevs so as long they're not being created and destroyed in quick succession it should be fine.

Reply:
Apparently 3 might not be enough as the latest CI build still failed the `core_4` test. I'm going to increase it to 10.